templates/alarms/content/alarms.tf

```
@@ -40,8 +40,8 @@ resource "aws_cloudwatch_log_metric_filter" "filter_system_warning" {
 }
 
 resource "aws_cloudwatch_metric_alarm" "alarm_system_warning" {
-  alarm_name          = ${{ values.warning_alarm_name | dump }} 
-  alarm_description   = ${{ values.warning_alarm_description | dump }} 
+  alarm_name          = ${{ values.service_name }}-Warning
+  alarm_description   = ${{ values.service_name }} warnings >= ${{ values.warning_threshold }} in 1 minute
```

Same comment about quotes:
```suggestion
  alarm_name          = "${{ values.service_name }}-Warning"
  alarm_description   = "${{ values.service_name }} warnings >= ${{ values.warning_threshold }} in 1 minute"
```